http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -166,10 +168,12 @@ protected String getRedirectUri(String state) {
 
             List<String> forwardableQueryParams = Arrays.asList(LOGIN_HINT, DOMAIN_HINT, KC_IDP_HINT, PROMPT, MAX_AGE, UI_LOCALES, SCOPE);
             List<NameValuePair> forwardedQueryParams = new ArrayList<>(forwardableQueryParams.size());
+            Set<String> allScopes = addScope(deployment.getScope(), new HashSet<>());
```

Comment:
Just a minor comment, readability wise, it would be good to initialize allScopes and then call addScope, i.e.,

Set<String> allScopes = new HashSet<>();
addScope(deployment.getScope(), allScopes);

Reply:
Fixed!